bin/tag-and-release.sh

```diff
@@ -22,6 +22,10 @@ function parse_flags() {
                 outdated="1"
                 shift
                 ;;
+            --commit-id=*)
+                commit_id="${1#*=}"
```

Comment:
Any reason to automatically set outdated to 1 here or do you think it's better to make the user explicitly pass outdated? If the latter let's make sure to document it.

Reply:
Surfacing the warning that the git tree is "dirty" is worthwhile IMO. If the user wants to override this behavior they will have to call the script directly: `./bin/tag-and-release.sh --commit-id=<SHA> --outdated`.

Reply:
can you add all this to the readme https://github.com/replicatedhq/kURL/blob/main/README.md#releases